### PR TITLE
Make sure Parameters Expressions are string when saving custom styles

### PR DESCRIPTION
**Pull request by frapell**
This PR fixes gh-209

### CODE REVIEW

**Comment by frapell** [comment]
@jenkins-plone-org please run jobs

File: src/plone/app/theming/browser/controlpanel.py [hunk 1]
```diff
@@ -188,6 +200,8 @@ def update(self):
 
             for line in parameterExpressionsList:
                 try:
+                    if isinstance(line, six.binary_type):
```
Comment (mauritsvanrees) [inline src/plone/app/theming/browser/controlpanel.py:203]: I have tried this on standard Plone 5.2 on Py 2 and 3. The change works. But I don't completely trust it for Python 2. If the line is binary, you decode it to unicode, but afterwards the code calls `str` on it, making it binary again. In fact when I try it in Python 2, the line is already unicode, so this change has no effect. But I still don't trust it. Please do this on Python 3 only:

```suggestion
                    if six.PY3 and isinstance(line, six.binary_type):
```

**Review by mauritsvanrees — CHANGES_REQUESTED** [review]
It works, and should be fine, but I don't completely trust it on Python 2. See inline comment.

A bit related is https://github.com/plone/Products.CMFPlone/issues/1759 which I saw today. Lesson from there is that currently `parameterExpressions` keys and values must be ASCII.

**Comment by frapell** [comment]
@mauritsvanrees Thank you for your suggestion, applied.

@jenkins-plone-org please run jobs

**Review by mauritsvanrees — APPROVED** [review]

**Pull request merged**